CHANGELOG.rst

```diff
@@ -4,6 +4,22 @@ Dellemc.Powerscale Change Logs
 
 .. contents:: Topics
 
+
+v3.1.0
+======
+
+Minor Changes
+-------------
+
+- Added support for OneFS 9.8 Lonestar release.
+- Added support for running as root, along with other boolean flags such as allow_delete_readonly, allow_execute_always, and inheritable_path_acl.
```

Comment:
This is for SMB share module, please mention that.

Reply:
yes updated